.circleci/config.yml

```diff
@@ -513,6 +513,21 @@ jobs:
             bundle exec fastlane android verify_debug_view_snapshot_tests
       - android/save-build-cache
 
+  emerge_purchases_ui_snapshot_tests:
+    description: "Emerge purchases ui snapshot tests"
+    <<: *android-executor
+    steps:
+      - checkout
+      - install-sdkman
+      - revenuecat/install-gem-unix-dependencies:
+          cache-version: v1
+      - android/restore-build-cache
+      - run:
+          name: Build and run purchases ui snapshot tests
+          command: |
+            bundle exec fastlane android emerge_purchases_ui_snapshot_tests
```

Comment:
I was gonna comment on the naming of the CircleCI job and Fastlane lane, as the Gradle module is called `revenuecatui`, and we're calling it "purchases ui" here, but I see  that the existing test app already has "purchasesui" in the name, so that inconsistency is not new haha.

Reply:
Yeah that's a good comment... I think we can fix separately to make it consistent 👍